projects/ngx-prx-styleguide/src/lib/charts/category-chart.component.ts

```diff
@@ -4,18 +4,19 @@ import { CategoryChartModel } from './models/category-chart.model';
 
 @Component({
   selector: 'prx-category-chart',
-  template: `<div #chart></div>`,
+  template: `
+    <div #chart></div>
+  `,
   styleUrls: ['./chart.css']
 })
-
 export class CategoryChartComponent implements OnChanges {
   @Input() data: CategoryChartModel[];
   @Input() color = '#0089bd';
   @Input() rotated = true;
   @Input() dataLabel = 'amount';
 
   chart: any;
-  @ViewChild('chart') el: ElementRef;
+  @ViewChild('chart', { static: true }) el: ElementRef;
```

Comment:
Angular 8 ViewChild needs a static property. `static: true` if the element is not bound by ngIf or ngFor and will be available on init; otherwise `static: false`